src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderTransactionHistoryDto.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Builder;
+import lombok.ToString;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDate;
+
+@ApiModel(description = "Offender transaction details")
+@Data
+@EqualsAndHashCode
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OffenderTransactionHistoryDto {
+
+    @ApiModelProperty(value = "Offender Id", example = "1", position = 1)
+    private Long offenderId;
+
+    @ApiModelProperty(value = "Transaction Id", example = "1", position = 2)
+    private Long transactionId;
+
+    @ApiModelProperty(value = "Transaction Sequence", example = "1", position = 3)
+    private Long transactionEntrySequence;
+
+    @ApiModelProperty(value = "Transaction Date", example = "2020-12-11", position = 4)
+    private LocalDate entryDate;
+
+    @ApiModelProperty(value = "Transaction Type", example = "???", position = 5)
```

Comment:
Just a reminder to fill in the missing examples when you can (and for referenceNumber and accountType)